src/Integration.Vsix/CFamily/VcxProject/FileConfigProvider.cs

```diff
@@ -45,6 +46,7 @@ internal class FileConfigProvider(
         IThreadHandling threadHandling) : IFileConfigProvider
     {
         private static readonly NoOpLogger noOpLogger = new NoOpLogger();
+        public IFileSystem FileSystem { get; set; } = new FileSystem();
```

Comment:
There's now a mef-importable IFileSystemService that you can use instead of manually creating the class here